src/components/views/projects/services.ts

```diff
@@ -50,14 +52,15 @@ export const fetchProjects = async (
 				: getMainCategorySlug(selectedMainCategory),
 			qfRoundSlug: isArchivedQF ? routerQuerySlug : null,
 		},
+		fetchPolicy: 'no-cache',
```

Comment:
we set "no cache" for testing purposes?

Reply:
@divine-comedian it is Apollo fetch, we're using React Query to fetch data and it cache those data and also BE will cache it when you try re-fetch. But those cache is about 2-3 minutes.